presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/DbResourceGroupsModule.java

```diff
@@ -27,10 +29,11 @@
     @Override
     public void configure(Binder binder)
     {
-        configBinder(binder).bindConfig(DbResourceGroupConfig.class);
+        configBinder(binder).bindConfig(DbReloadingResourceGroupConfig.class);
+        binder.bind(ReloadingResourceGroupConfig.class).to(DbReloadingResourceGroupConfig.class).in(Scopes.SINGLETON);
         binder.bind(ResourceGroupsDao.class).toProvider(MysqlDaoProvider.class).in(Scopes.SINGLETON);
-        binder.bind(DbResourceGroupConfigurationManager.class).in(Scopes.SINGLETON);
-        binder.bind(ResourceGroupConfigurationManager.class).to(DbResourceGroupConfigurationManager.class).in(Scopes.SINGLETON);
-        newExporter(binder).export(DbResourceGroupConfigurationManager.class).withGeneratedName();
+        binder.bind(ReloadingResourceGroupConfigurationManager.class).in(Scopes.SINGLETON);
+        binder.bind(ResourceGroupConfigurationManager.class).to(ReloadingResourceGroupConfigurationManager.class).in(Scopes.SINGLETON);
+        newExporter(binder).export(ReloadingResourceGroupConfigurationManager.class).withGeneratedName();
```

Comment:
The idea of introducing `ReloadingResourceGroupConfigurationManagerModule` is to ensure that all components in the `reloading` module are initiated via the `ReloadingResourceGroupConfigurationManagerModule` rather than other modules implementing it.

This module should just be focusing on the components that are in the DB module (this is my guess) - 

```
configBinder(binder).bindConfig(DbReloadingResourceGroupConfig.class);
binder.bind(ResourceGroupsDao.class).toProvider(MysqlDaoProvider.class).in(Scopes.SINGLETON);
binder.bind(ManagerSpecProvider.class).to(DbManagerSpecProvider.class).in(Scopes.SINGLETON);
```